tutorials/group_level_analysis/plot_group_level_with_individual_clusters.py

```diff
@@ -94,5 +94,16 @@
     d["subject_id"] = subject_id
     ms_data.append(d)
 
-ms_data = pd.DataFrame(ms_data)
-ms_data
+#%%
+# From this point on, we can start to visualize our results
+# and do your statistical analysis. For example we can
+# plot the :term:`GEV` of microstate class.
+
+import matplotlib.pyplot as plt
+
+data = np.array([[d['MS1_gev'], d['MS2_gev'], d['MS3_gev'], d['MS4_gev'], d['MS5_gev']] for d in ms_data])
```

Comment:
Yes, because they do not really look like usual "classical"  MS topographies. I don't want people to miss interpret topographies because they saw the name on one of our tutorials.

Reply:
Ok!